### PR TITLE
Make Function state valid values explicit

### PR DESCRIPTION
**Pull request by janelletavares**
Most of the changes were redundant... leaving the remaining changes in this PR to make setting function status clearer.

### CODE REVIEW

File: pkg/meroxa/api_errors_test.go [hunk 1]
```diff
@@ -22,14 +22,6 @@ func errorJSONResponse(msg string) io.ReadCloser {
 // create a new reader with an HTML response
 var errorHTMLResponse = ioutil.NopCloser(bytes.NewReader([]byte(`<h1>Error!</h1>`)))
 
-var http503JSONResponse = &http.Response{
```
Comment (raulb) [inline pkg/meroxa/api_errors_test.go:25]: I'm assuming this wasn't used anywhere?

**Review by raulb — COMMENTED** [review]
@janelletavares I'm not sure I recall the motivation behind this change (nothing in the PR description). I'll dig if I remember where this is coming from so I feel more confident approving / commenting on this PR.

**Comment by lyuboxa** [comment]
@janelletavares is this duplicate of https://github.com/meroxa/meroxa-go/pull/118?

File: pkg/meroxa/api_errors_test.go (hunk 1 above)
Comment (janelletavares) [reply pkg/meroxa/api_errors_test.go]: it wasn't... i ran lint to find that, iirc

**Comment by nassor** [comment]
Still a valid change?

**Comment by janelletavares** [comment]
> Still a valid change? (waiting for review)

@nassor yup, still useful, though not urgent/related to any open tickets

**Review by samirketema — APPROVED** [review]

**Review by nassor — APPROVED** [review]

**Pull request merged**